apps/schools/domains/user/components/auth/containers/Cookies.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import styles from '@domains/user/components/auth/containers/styles/styles.module.scss'
+import Image from 'next/image'
+import arrowBottom from '@public/icons/arrowBottom.svg'
+import arrowTop from '@public/icons/arrowTop.svg'
+import { Button } from '@domains/common/components/button'
+import { Typography } from 'antd'
+import Cookie from 'universal-cookie'
+
+const COOKIE_AGREEMENT_KEY = 'cookieAgreementAccepted'
+
+export const Cookies = () => {
+    const cookies = new Cookie()
+
+    const [isShown, setIsShown] = useState(false)
+    const [cookiesNotAccepted, setCookiesNotAccepted] = useState(true)
+
+    const acceptCookieAgreement = useCallback(() => {
+        cookies.set(COOKIE_AGREEMENT_KEY, 'true')
+        setCookiesNotAccepted(false)
+    }, [])
+
+    useEffect(() => {
+        const cookieValue = cookies.get(COOKIE_AGREEMENT_KEY)
+        if (cookieValue) {
+            setCookiesNotAccepted(false)
+        }
+    }, [])
+
+    if (!cookiesNotAccepted) {
```

Comment:
did you find some references what we should do if user refuse to use cookies?
should we check everywhere using cookie this variable?

Reply:
I do not understand at all why these popups are with the word "accept" (the policy is meant) anyway cookies are used

Reply:
The doma.ai have this popup, but they always use cookies